lib/topic.ts

```diff
@@ -3,11 +3,11 @@ import { showElementActions } from './kafka';
 import { navigateToKafkaList } from './navigation';
 
 export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
-  await expect(page.getByText(kafkaName)).toHaveCount(1);
-  await page.locator('a', { hasText: kafkaName }).click();
-  await expect(await page.locator('button[aria-label="Topics"]')).toHaveCount(1);
+  await expect(page.getByText(`${kafkaName}`)).toHaveCount(1);
+  await page.locator('a', { hasText: `${kafkaName}` }).click();
```

Comment:
Why this change? Is there some difference in variable evaluation? I didn't find it in pw doc

Reply:
It is to tighten that the String value of the variable is used, to make sure

Reply:
`kafkaName` has type `string`, can you elaborate more when it wasn't string?

Reply:
both ways resolve to a String, maybe this one is more robust. The same `${}` is used throughout the project @mmusil Can you confirm, please?

Reply:
Where it is used? Could you share some docs where the robustness is described?

Reply:
Can I resolve this conversation?

Reply:
I still don't see any prove that your new code is more robust than the old one

Reply:
As I mentioned above the same way it is being used throughout the project, I triggered tests, they are passing twice in a row: 
<img width="1412" alt="image" src="https://user-images.githubusercontent.com/100594859/218498738-7e2d329b-2926-4cc5-b46e-0c0a2cd1a717.png">
<img width="1421" alt="image" src="https://user-images.githubusercontent.com/100594859/218498903-4a17ed1d-37fa-45f3-bc0c-e2ed29760ce8.png">

Reply:
This still doesn't explain the need for this change. We should follow one pattern. Anyway, I guess I can live with it...